nbresuse/prometheus.py

```diff
@@ -61,3 +69,44 @@ def apply_cpu_limit(self, cpu_metric_values) -> Optional[float]:
                 return self.config.cpu_limit
             else:
                 return 100.0 * cpu_metric_values["cpu_count"]
+            
+    @gen.coroutine
+    def kernel_metrics(self):
+        kernels = yield maybe_future(self._list_kernel_memory())
+        for kernel_id, data in kernels.items():
+            self.KERNEL_MEMORY_USAGE.labels(kernel_id=kernel_id).set(data['rss'])
+
+    @gen.coroutine
+    def _list_kernel_memory(self):
+        kernel_memory = dict()
+
+        session_manager = self.session_manager
+        sessions = yield maybe_future(session_manager.list_sessions())
+        kernel_processes = PrometheusHandler._kernel_processes()
+
+        def find_process(session_id):
```

Comment:
Should the parameter be named `session_id` or `kernel_id`? (to keep the same naming convention)

Reply:
Yeah I will change to `kernel_id`